components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -517,6 +517,8 @@ public static class JSAttributes {
         public static final String FORCE_AUTH_PARAM = "forceAuth";
         public static final String SUBJECT_IDENTIFIER_PARAM = "markAsSubjectIdentifierStep";
         public static final String SUBJECT_ATTRIBUTE_PARAM = "markAsSubjectAttributeStep";
+        public static final String AUTHENTICATOR_NAME_IN_AUTH_CONFIG =
+                "AdaptiveAuth.AuthenticatorNameInAuthConfig.Enable";
```

Comment:
use the AdaptiveAuthentication inner class

Reply:
Fixed